### PR TITLE
TRANSFER-544: airflow tf-provider implementation

### PR DESCRIPTION
**Pull request by kemalkochekov**

### CODE REVIEW

File: internal/provider/airflow_cluster_resource.go [hunk 1]
```diff
@@ -0,0 +1,870 @@
...
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"name": schema.StringAttribute{
+				Required:            true,
+				MarkdownDescription: "Cluster name",
+			},
+			"description": schema.StringAttribute{
+				Optional:            true,
+				Computed:            true,
```
Comment (laskoviymishka) [inline internal/provider/airflow_cluster_resource.go:656]: why it's computed?

File: internal/provider/airflow_cluster_resource.go [hunk 2]
```diff
@@ -0,0 +1,870 @@
...
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"connection_info": schema.SingleNestedAttribute{
+				Computed:            true,
+				Attributes:          airflowConnectionInfoResSchema(),
+				PlanModifiers:       []planmodifier.Object{objectplanmodifier.UseStateForUnknown()},
+				MarkdownDescription: "Public connection info",
+			},
+			//"custom_remote_connection_info": schema.SingleNestedAttribute{
```
Comment (laskoviymishka) [inline internal/provider/airflow_cluster_resource.go:680]: maybe better to delete it?

File: internal/provider/airflow_cluster_resource.go [hunk 3]
```diff
@@ -0,0 +1,870 @@
...
+			"config": schema.SingleNestedBlock{
+				Description: "Cluster configuration",
+				Attributes: map[string]schema.Attribute{
+					"version_id": schema.StringAttribute{
+						Required:            true,
+						MarkdownDescription: "Version ID of the Airflow cluster.",
+					},
+					"custom_image_digest": schema.StringAttribute{
+						Optional:            true,
+						MarkdownDescription: "Custom image digest for the Airflow cluster.",
```
Comment (laskoviymishka) [inline internal/provider/airflow_cluster_resource.go:753]: this 2 fields acts as one-of, it's either custom image or requirement txt, let's add this do comment.

**Review by laskoviymishka — COMMENTED** [review]
overral LGTM, but tests red

File: internal/provider/airflow_cluster_resource_test.go [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+      dags_path = %[15]q
+    }
+  }
+
+  access {
+    data_services = ["transfer"]
+    ipv4_cidr_blocks {
+      value       = "10.0.0.0/8"
+      description = "Office in Berlin"
+    }
```
Comment (ivoninskii) [inline internal/provider/airflow_cluster_resource_test.go:125]:
```suggestion
    ipv4_cidr_blocks [
      {
        value       = "10.0.0.0/8"
        description = "Office in Berlin"
      }
    ]
```

**Review by ivoninskii — APPROVED** [review]

**Review by laskoviymishka — APPROVED** [review]

**Pull request merged**